src/Components/SchemaDotOrg/dataFromJob.js

```diff
@@ -0,0 +1,41 @@
+import formatDate from 'utils/formatDate';
+import textExtractFromObj from 'utils/textExtractFromObj';
+
+function dataFromJob(job) {
+  return {
+    '@context': 'http://schema.org',
+    '@type': 'JobPosting',
+    title: job.get('title'),
+    datePosted: formatDate(job.get('datePosted'), 'yyyy-mm-dd'),
+    validThrough: formatDate(job.get('validThrough'), 'yyyy-mm-dd'),
+    description: textExtractFromObj(job),
```

Comment:
Why not `textExtractFromWidgetlist(obj.get('body'))`? I know, that currently they are the same, but who knows, if someone might add the address to `registerTextExtract` of a job.